src/Http/Controllers/SwaggerController.php

```diff
@@ -72,6 +72,10 @@ public function docs(Request $request, string $file = null)
             }
         }
 
+        if (! File::exists($filePath)) {
```

Comment:
But the file should be created on line 60 and if not we have an error for that. What was a situation you encountered that file was not created?

Reply:
Hi, just try to access the json file directly to browser, then try to input some not existing file will return 500 status

Reply:
Ah, that makes sense. Can you add the error message something like `Documentation not found`? And also add the test for this?

Reply:
Yes I can, I'll be right back :)